client.go

```diff
@@ -333,6 +333,9 @@ func (c *Client) cleanup(ch chan *Event) {
 
 func trimHeader(size int, data []byte) []byte {
 	data = data[size:]
```

Comment:
good point on this. what if data is nil or len(data) is < size?

Reply:
Good question. I think that usecase is covered by https://github.com/r3labs/sse/blob/master/client.go#L283 and the `bytes.HasPrefix` checks before its invoked.